test/plugin/test_buf_file.rb

```diff
@@ -1232,44 +1230,85 @@ def compare_log(plugin, msg)
       assert { logs.any? { |log| log.include?(msg) } }
     end
 
-    test '#resume ignores staged empty chunk' do
-      _, p1 = create_first_chunk('b')
+    test '#resume backups staged empty chunk' do
+      setup_plugins({'path' => @bufpath})
+      c1id, p1 = create_first_chunk('b')
       File.open(p1, 'wb') { |f| } # create staged empty chunk file
       c2id, _ = create_second_chunk('b')
 
-      @p.start
+      Fluent::SystemConfig.overwrite_system_config('root_dir' => @bufdir) do
+        @p.start
+      end
+
       compare_staged_chunk(@p.stage, c2id, '2016-04-17 14:01:00 -0700', 3, :staged)
       compare_log(@p, 'staged file chunk is empty')
+      assert_false File.exist?(p1)
```

Comment:
Power assert style (`assert { File.exist?(p1) }`) is preferred:
https://github.com/fluent/fluentd/pull/4034#discussion_r1096937376

Reply:
Thanks! Yes! Power assertion is great! I have fixed entirely assertions related to this fix.